frontend/pages/SoftwarePage/SoftwareTitleDetailsPage/ViewYamlModal/ViewYamlModal.tsx

```diff
@@ -83,6 +85,7 @@ const ViewYamlModal = ({
     postInstallScript,
     uninstallScript,
     iconUrl: iconUrl || null,
+    displayName: displayName || undefined,
```

Comment:
Should be "" I think, which would match the API response. We still shouldn't show display name in that case.